### PR TITLE
[atk][exiv2][gdk-pixbuf][gettext][glib][libcroco][tesseract] Replace uses of unofficial-gettext with FindIntl

### PR DESCRIPTION
**Pull request by ras0219**
See title. See also #13027.

We prefer supporting built-in cmake modules whenever possible instead of providing our own replacements.

### CODE REVIEW

**Comment by strega-nil** [comment]
I think this might've been fixed already by @Neumann-A in #11776, no?

**Comment by PhoebeHui** [comment]
After compared with PR https://github.com/microsoft/vcpkg/pull/11776, the changes for atk and gettext still needed, I will help to undo changes for other ports and resolve the conflicts.

File: ports/gettext/portfile.cmake [hunk 1]
```diff
@@ -59,24 +59,21 @@ else()
...
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
-
-set(GNU_DLL_PATHS lib/ debug/lib/)
-set(GNU_DLL_NAME GNU.Gettext.dll) #C# dll?
-foreach(DLL_PATH IN LISTS GNU_DLL_PATHS)
-    if(EXISTS "${CURRENT_PACKAGES_DIR}/${DLL_PATH}${GNU_DLL_NAME}")
-       file(REMOVE "${CURRENT_PACKAGES_DIR}/${DLL_PATH}${GNU_DLL_NAME}")
-    endif()
+# These targets are provided for backwards compatibility with previous vcpkg versions
+vcpkg_fixup_cmake_targets(CONFIG_PATH share/unofficial-gettext TARGET_PATH share/unofficial-gettext)
```
Comment (PhoebeHui) [inline ports/gettext/portfile.cmake:63]: The CI failed with following errors:

```
Make Error at scripts/cmake/vcpkg_fixup_cmake_targets.cmake:126 (message):
  'D:/packages/gettext_x64-windows-static/debug/share/unofficial-gettext'
  does not exist.
Call Stack (most recent call first):
  ports/gettext/portfile.cmake:63 (vcpkg_fixup_cmake_targets)
  scripts/ports.cmake:136 (include)
```

**Review by PhoebeHui — CHANGES_REQUESTED** [review]
@ras0219, I think we need check if unofficial-gettext exist.

File: ports/gettext/portfile.cmake (hunk 1 above)
Comment (Neumann-A) [reply ports/gettext/portfile.cmake:63]: it does not exists anymore since gettext was changed to native build which is make based

**Pull request closed without merging**